src/calculator/service.ts

```diff
@@ -61,8 +61,25 @@ function getBadWords(words: string[], lang: Language) : string[] {
   return words.filter(isBadWord(lang))
 }
 
+function removeURL(text: string) {
+  return text.replace(/(https?:\/\/[^\s]+)/g,'')
+}
+
+function removeMention(text: string) {
+  return text.replace(/\B@[a-z0-9_-]+\s/gi,'')
+}
+
+function removePunctuation(text: string) : string{
+  return text.replace(/(~|`|!|@|#|$|%|^|&|\*|\(|\)|{|}|\[|\]|;|:|"|'|<|,|\.|>|\?|\/|\\|\||-|_|\+|=)/g,'')
+}
+
+function cleanTweetText(text: string) : string{
```

Comment:
I think that we should rename this functions since we are not only cleaning tweet texts. Something like `cleanText`

Reply:
done